src/include/parser/expression/parsed_function_expression.h

```diff
@@ -35,6 +35,11 @@ class ParsedFunctionExpression : public ParsedExpression {
         : ParsedExpression{expressionType_}, isDistinct{isDistinct},
           functionName{std::move(functionName)} {}
 
+    ParsedFunctionExpression(std::string functionName, std::vector<std::string> optionalArguments,
```

Comment:
Can u use `addOptionalParams`. I kinda wanna reduce the number of constructors for this class.

Reply:
Not really. I'd have to add a new function to set `optionalArguments` only.